packages/web-components/playwright.config.ts

```diff
@@ -31,10 +32,10 @@ const config: PlaywrightTestConfig<TestOptions> = {
              * Increases the maximum allowed pixel difference to account
              * for slight browser rendering inconsistencies.
              */
-            // maxDiffPixelRatio: 0.01,
+            maxDiffPixelRatio: 0.02,
```

Comment:
I've updated these because rux-clock's time would change during the VRT's, and thus they'd fail by 0.01% of pixels being off.